CI Fail:
```
AssertionError: ["In file '''/home/runner/work/rosdistro/rosdistro/noetic/distribution.yaml''': 
Could not validate source entry for repo network_interface with error 
[[[Could not validate repository with url https://github.com/astuff/network_interface.git and version 3.0.0 from entry at line 4217.
Error reason: Tags are not valid, but a tag 3.0.0 was found. Re: https://github.com/ros/rosdistro/pull/20286]]]"] is not false
```

I don't know how it was able to find a 3.0.0 tag but then determine it to be invalid?

--- a/noetic/distribution.yaml
+++ b/noetic/distribution.yaml
@@ -4208,7 +4208,7 @@ repositories:
     doc:
       type: git
       url: https://github.com/astuff/network_interface.git
-      version: 3.0.0
+      version: master
     release:
       tags:
         release: release/noetic/{package}/{version}
@@ -4217,7 +4217,7 @@ repositories:
     source:
       type: git
       url: https://github.com/astuff/network_interface.git
-      version: 3.0.0
+      version: master
     status: maintained
   nmea_comms:
     doc:

